third_party/metrics_proto/METADATA

```diff
@@ -0,0 +1,17 @@
+name: "metrics_proto"
+description:
+  "These are the proto definitions to upload metrics data to Google."
+
+third_party {
+  url {
+    type: GIT
+    value: "https://chromium.googlesource.com/chromium/src/third_party/metrics_proto/"
+  }
+  version: "fd3a73980b82eea000403c2c9c1f204f6612e7cf"
+  last_upgrade_date {
+    year: 2018
+    month: 8
+    day: 30
+  }
+  license_type: NOTICE
+}
```

Comment:
tiny nit: looks like missing line end. Our pre-submits do not check anything in third_party/ by default, so it wouldn't alert on it.

Reply:
Done